101-cosmosdb-api-for-mongodb-account-create/azuredeploy.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Hi @bmoore-msft, I just doubled check and the 3 .json files are already well-formatted within Visual Studio Code. Nothing to commit after doing Alt+Shift+F on them.